packages/odyssey-react/src/components/Table/Table.theme.ts

```diff
@@ -13,5 +13,5 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
```

Comment:
hmmm I think we should refactor further to get the effect we want. I notice that we *require* the `caption` prop, we should rename that to something like `screenReaderCaption`  and allow `heading` to become `caption`. If the screen reader content was *not* required I would suggest a single prop `caption` that is typed as `ReactNode` to allow consumers of our API to put any combination of visible or SR only text there providing more flexibility via composition at the expense of someone perhaps not including SR text when they should?

Reply:
I like your first proposal of `screenReaderCaption` and `caption`. The `caption` element is always hidden (today) and only for SR purposes, so that's a more informative prop name.